install.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+DIRECTORY="fcc-python-vagrant"
+
+echo "This script requires superuser access to install apt packages."
+echo "You will be prompted for your password by sudo."
+
+# clear any previous sudo permission
+sudo -k
+
+# run inside sudo
+sudo sh <<SCRIPT
+
+echo "Installing VirtualBox ..."
+
+if hash virtualbox 2>/dev/null; then
+	echo "Virtualbox already installed, skipping ..."
+else
+	wget -q https://www.virtualbox.org/download/oracle_vbox_2016.asc -O- | apt-key add -
+	wget -q https://www.virtualbox.org/download/oracle_vbox.asc -O- | apt-key add -
+
+	apt-get update -y
+	apt-get install -y virtualbox
+fi
+
+echo "Installing dkms ..."
+if hash dkms 2>/dev/null; then
+	echo "Dkms already installed, skipping ..."
+else
+	apt-get install -y dkms
+fi
+
+echo "Installing VirtualBox extensions ..."
+
+wget -O /tmp/Linux_amd64.run http://download.virtualbox.org/virtualbox/5.0.20/VirtualBox-5.0.20-106931-Linux_amd64.run
+
+chmod +x /tmp/Linux_amd64.run
+./tmp/Linux_amd64.run
```

Comment:
I ran the script once after removing `.`

Here's the screenshot

![screenshot from 2016-05-14 15 51 57](https://cloud.githubusercontent.com/assets/5607371/15267901/0fd7148a-19ec-11e6-80bd-2542ac14eb6e.png)

Notice the error 

```bash
/tmp/Linux_amd64.run: 311: exec: -title: not found
```

What does that install? The extension pack?